sdk/batch/batch/sample.env

```diff
@@ -0,0 +1,12 @@
+# Used in most samples. Retrieve these values from a Batch instance
+# in the Azure Portal.
+AZURE_BATCH_ENDPOINT="<Batch Endpoint>"
+AZURE_BATCH_ACCOUNT_NAME="<resource name>"
+AZURE_BATCH_ACCOUNT_KEY="<account key>"
+# Used to authenticate using Azure AD as a service principal for role-based authentication
+# in the tokenAuth sample.
+#
+# See the documentation for `EnvironmentCredential` at the following link:
+# https://docs.microsoft.com/javascript/api/@azure/identity/environmentcredential
+AZURE_CLIENT_ID="<application id>"
+AZURE_CLIENT_SECRET="<application secret>"
```

Comment:
Can we move this file to the test folder? This way when we re-generate the package, we can assume that everything under the test folder is hand written, everything else is auto generated

Reply:
done.